racket/src/cs/c/configure.ac

```diff
@@ -611,9 +639,11 @@ fi
 
 ############## other inferred machine properties ################
 
-AC_CHECK_SIZEOF([void *])
+if test "${enable_ios}" = "" ; then
```

Comment:
I'm not sure why this check fails yet and it would be good to get it to work before merging. The size that is detected is 0 and I'm not sure why.